site/AntDesign.Docs/Demos/Components/Table/demo/Dynamic.razor

```diff
@@ -1,22 +1,35 @@
 ﻿@using System.ComponentModel
 @using AntDesign.TableModels
 
-<Table TItem="Dictionary<string, object>" OnChange="HanleChange" DataSource="@data" Loading="data==null" ScrollX="1500" PageSize="5" Size="TableSize.Small">
-  @foreach (var key in data?.FirstOrDefault()?.Keys.Take(10) ??new string[0])
-  {
-    <PropertyColumn Property=@(c=>c[key]) Title="@key"></PropertyColumn>
-  }
+<Table TItem="Dictionary<string, object>" OnChange="HanleChange" DataSource="@data" Loading="data==null" ScrollX="1500"
+  PageSize="5" Size="TableSize.Small">
+    @foreach (var key in data?.FirstOrDefault()?.Keys.Take(10) ?? new string[0])
+    {
+        <PropertyColumn Property=@(c=>c[key]) Title="@key"></PropertyColumn>
+    }
 </Table>
 
 @inject HttpClient httpClient;
 @code {
 
-  List<Dictionary<string, object>> data;
+    List<Dictionary<string, object>> data;
 
-  string githubUrl = "https://api.github.com/repos/ant-design-blazor/ant-design-blazor/contributors?per_page=200";
+    string githubUrl = "https://api.github.com/repos/ant-design-blazor/ant-design-blazor/contributors?per_page=200";
 
-  protected async Task HanleChange(QueryModel<Dictionary<string, object>> query)
-  {
-    data = await httpClient.GetFromJsonAsync<List<Dictionary<string, object>>>(githubUrl);
-  }
-}
+    protected async Task HanleChange(QueryModel<Dictionary<string, object>> query)
+    {
+        var environment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+        if (environment == "Development")
+        {
+            httpClient.DefaultRequestHeaders.Add("User-Agent", "BlazorApp");
```

Comment:
I think we can just add this header, which will reduce the complexity of this example. We believe that the simpler the example, the better!

Reply:
Sure, you are right I will remove unnecessary codes.
But what about Try/Catch ?
Should I remove It too ?

Reply:
> I think we can just add this header, which will reduce the complexity of this example. We believe that the simpler the example, the better!

Sure, you are right I will remove unnecessary codes.
But what about Try/Catch ?
Should I remove It too ?

Reply:
Yes, it's better that add a DelegatingHandler for the HttpClient to handle the headers and exceptions, and this is out of the scope of the exsample.

Reply:
> I think we can just add this header, which will reduce the complexity of this example. We believe that the simpler the example, the better!

Done.

Reply:
How about using DelegatingHandler?

Reply:
> How about using DelegatingHandler?

Do you mean adding a new class for http client then register as DI in Program.cs and using it in component ?

Reply:
Yes, but please add it in the `AntDesign.Docs.Server`, and the finally using is in this example. So that the example is no need to be changed. 

Sorry I didn't think of this scenario until later.

Reply:
> Yes, but please add it in the `AntDesign.Docs.Server`, and the finally using is in this example. So that the example is no need to be changed.
> 
> Sorry I didn't think of this scenario until later.

I'll do it ASP.

Reply:
> Yes, but please add it in the `AntDesign.Docs.Server`, and the finally using is in this example. So that the example is no need to be changed.
> 
> Sorry I didn't think of this scenario until later.
@ElderJames 
That's done now, Please check commits again.